docs/reference/upgrade/disable-shard-alloc.asciidoc

```diff
@@ -17,3 +17,5 @@ PUT _cluster/settings
 }
 --------------------------------------------------
 // TEST[skip:indexes don't assign]
+
+You may also consider <<modules-gateway,gateway settings>>. 
```

Comment:
this sentence doesn't make the connection to gateway settings clear. consider:

```suggestion
As an alternative to disabling replica allocation, you can adjust your <<modules-gateway,gateway settings>> to [XYZ action]. 
```

Reply:
> this sentence doesn't make the connection to gateway settings clear. consider:

That's probably ignorance on my part, sorry. This setting would not be in replacement but in addition to. I believe you would also use gateway settings if your start-up times between nodes were going to be a bit delayed compared to normal and/or to reduce initial strain (from allocation kicking off) while doing discovery.

Reply:
what setting config would you expect them to use in a full cluster restart scenario?

Reply:
IME when users set one they set all three at once. But I guess if we were going to link to only one, it'd be `gateway.recover_after_data_nodes`.

Reply:
can you edit this sentence to reflect that they also might want to set these things due to the reasons you explained? we can link to the top of the settings page - I just didn't understand the connection between these two topics

Reply:
Updated, cheers.